clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -1881,6 +1882,8 @@ protected void processCommonConsumerAction(
           startConsumingAsLeader(newPartitionConsumptionState);
         } else {
           updateLeaderTopicOnFollower(newPartitionConsumptionState);
+          reportUserStoreOffsetRewindMetrics(newPartitionConsumptionState);
```

Comment:
Looks like we are only doing this rewind check for version topics and not RT topics. Is this the complete scope of the ticket/fix? If so we should probably make it clear in the description or commit message.

Reply:
Thanks Xun for the review. You are right, this change is targeting the VT topic only. For offset rewind issues in RT topic, I assume it has been taken care by LeaderFollowerStoreIngestionTask.checkAndHandleUpstreamOffsetRewind. Yeah, I need to update the description and make it more clear.